tutorials/scripting/c_sharp/c_sharp_exports.rst

```diff
@@ -0,0 +1,354 @@
+.. _doc_c_sharp_exports:
+
+C# Exports
+==========
+
+Introduction to exports
+-----------------------
+
+In Godot, class members can be exported. This means their value gets saved along
+with the resource (such as the :ref:`scene <class_PackedScene>`) they're
+attached to. They will also be available for editing in the property editor.
+Exporting is done by using the ``[Export]`` attribute::
+
+    public class ExportExample : Spatial
+    {
+        [Export]
+        public int Number = 5;
+    }
+    
+In that example the value ``5`` will be saved, and after building the current project
+it will be visible in the property editor. This way, artists and game designers
+can modify values that later influence how the program runs. For this, a
+special export syntax is provided.
+
+Exporting can only be done with built-in types or objects derived from the :ref:`Resource class <class_Resource>`.
+
+.. note::
+
+    Exporting properties can also be done in GDScript, for information on that
+    see :ref:`doc_gdscript_exports`.
+
+Basic use
+---------
+
+Exporting can work with and without setting a default value. For int and float
+``0`` will then be used as the default.
+
+::
+
+    [Export]
+	public int Number;
```

Comment:
After this line, every exported field is `private`. Maybe we should be consistent throughout the page?